doc/ref_cert/README.md

```diff
@@ -2,15 +2,14 @@
 
 # CNTT Reference Conformance (RC) Test Case Integration
 
-All CNTT conformance suites must fulfill the following test case integration
-requirements to guarantee a smooth overall integration, the same end user
-actions and a unique test result format (e.g. OPNFV test result database)
+All CNTT conformance suites must utilize the OPNFV test case integration
+toolchain to deliver overall integration, the same end user
```

Comment:
I'm sorry but I think the original proposal is better as it's about test case integration requirement (e.g. Docker, etc.).
why not to deliver instead to guarantee a smooth.

Reply:
https://github.com/cntt-n/CNTT/pull/1738#discussion_r459567392
This is from @pgoyal01 's comment

Reply:
If all agree, I would prefer the orig one.

Reply:
@collivier In the original what does "smooth" mean? What does "same end user actions" refer to?

Reply:
@pgoyal01 the original sentence is about the test case integration requirements (how the test cases are delivered, executed and their results published) rather than the upstream integration toolchain.
As it deeply simplifies the end user operations (no dependency management, no extra operations per test frameworks/suites, etc.), I considered the integration as smooth compared to the traditional issues when assembling sparsed test cases from different communities/projects.

Let's merge it if it pleases.

Reply:
@collivier The original sentence:
"All CNTT conformance suites must fulfill the following test case integration requirements to guarantee a smooth overall integration, the same end user actions and a unique test result format (e.g. OPNFV test result database) needed by the end users and the test case result verification programs (e.g. OVP)."


Here are the phrases with issues:
- "the following test case integration requirements": no such requirements are listed
- "a smooth overall integration": integration with what? also what does "smooth" mean?
- "the same end user actions":  same as what user actions?
- "a unique test result format": why unique?